.pipelines/templates/e2e-tests.yml

```diff
@@ -29,6 +29,7 @@ steps:
           python -m pip install --upgrade pip
           pip install -r requirements.txt
           pip install pytest-azurepipelines
+          python -m spacy download en_core_web_lg
```

Comment:
this will download spacy  module for the anonymizer as well

Reply:
Excatly

Reply:
Have you tried with the smaller mode? faster to download and run

Reply:
ok, scratch that - the e2e test run only once, not for every package. but still worth testing with the S size spacy model

Reply:
let's use the sm model for now